src/main/java/ollie/exception/OllieException.java

```diff
@@ -0,0 +1,10 @@
+package ollie.exception;
```

Comment:
good job placing all your exceptions in a package! applies to the other parts of your code as well